pages/landing/Footer.tsx

```diff
@@ -33,9 +33,9 @@ const Footer = () => {
           </div>
           {/* Social Brands */}
           <div className="mt-3 space-x-2">
-            <a
+            <Link
               className="inline-flex h-10 w-10 items-center justify-center rounded-full text-center text-gray-500 transition hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-600 focus:ring-offset-2 focus:ring-offset-white dark:text-gray-500 dark:hover:bg-gray-800 dark:hover:text-gray-200"
-              href="#"
+              href="https://github.com/danielcorreia-dev/edu-ai"
```

Comment:
You should put a redirect also put a _blank redirect on the Link component, try adding this: 
```suggestion
              href="https://github.com/danielcorreia-dev/edu-ai"
              target="_blank"
```

Reply:
Consertei.